### ISSUE TITLE
Error on wesher container bring-up

### ISSUE THREAD

**Issue by astew** [samizdapp/skunkworks#3]
When I run:

    balena push <id>.local

It does the docker-compose stuff mostly as expected, but eventually reaches a looping state where the log for the `wesher` container periodically (sometimes second sometimes more than a minute) errors and restarts, saying the command `/run.sh` is not found:

```
...
[Logs]    [3/20/2022, 6:03:37 PM] Restarting service 'wesher sha256:05487d7e9164e34c342697b1b3f18aa73341326e7db415b44060a8dc95dded53'
[Logs]    [3/20/2022, 6:03:37 PM] Starting service 'wireguard sha256:c459291d6e21dd1337ad26862e6f41c3cf6b5abec45ec28c7afa34ebd412051f'
[Logs]    [3/20/2022, 6:03:37 PM] [wesher] Command not found: /run.sh
[Logs]    [3/20/2022, 6:03:38 PM] Service exited 'wesher sha256:05487d7e9164e34c342697b1b3f18aa73341326e7db415b44060a8dc95dded53'
[Logs]    [3/20/2022, 6:03:40 PM] Restarting service 'wesher sha256:05487d7e9164e34c342697b1b3f18aa73341326e7db415b44060a8dc95dded53'
[Logs]    [3/20/2022, 6:03:39 PM] [wesher] Command not found: /run.sh
[Logs]    [3/20/2022, 6:03:40 PM] Service exited 'wesher sha256:05487d7e9164e34c342697b1b3f18aa73341326e7db415b44060a8dc95dded53'
[Logs]    [3/20/2022, 6:03:42 PM] Restarting service 'wesher sha256:05487d7e9164e34c342697b1b3f18aa73341326e7db415b44060a8dc95dded53'
[Logs]    [3/20/2022, 6:03:42 PM] [wesher] Command not found: /run.sh
[Logs]    [3/20/2022, 6:03:43 PM] Service exited 'wesher sha256:05487d7e9164e34c342697b1b3f18aa73341326e7db415b44060a8dc95dded53'
...
```

After a lot of digging (I don't have extensive experience with docker or balena) I found that `/run.sh` was being passed as an argument to the entry point script `/usr/bin/entry.sh` (set in an ancestor docker image), which was directly generating the error message after testing for the presence of `/run.sh` using `command -v /run.sh`.

I managed to get past this issue by adding commands to the wesher image's Dockerfile to `chmod a+x` each of the three scripts that get copied into the image:

```
...
COPY run.sh run.sh
COPY hosts_updated.sh .
COPY watch_hosts.sh .
RUN chmod a+x run.sh
RUN chmod a+x hosts_updated.sh
RUN chmod a+x watch_hosts.sh
CMD [ "/run.sh" ]
```

Don't know whether that's the appropriate fix, but hopefully it helps someone else avoid a lot of debugging.

**Comment by HOGGL** [comment]
Was having the same issues when trying to get this to run on my device. This was also a problem with the wireguard and matrix containers. It seems that @rynomad is either using a special version of docker or the Dockerfiles have fallen behind local development. This is fixed in #4.

**Comment by rynomad** [comment]
I think i did `chxmod` locally and made it work and naivelly thought that would propagate through git. I'll be mindful to make steps like that explicit in code in the future
